tools/internal_ci/helper_scripts/prepare_build_macos_rc

```diff
@@ -67,7 +67,8 @@ then
   time git clone --depth 1 https://github.com/CocoaPods/Specs.git ~/.cocoapods/repos/master
 
   # Needed for ios-binary-size
-  time pip install --user pyyaml pyjwt==1.7.1 pyOpenSSL cryptography requests
+  time pip install --user pyyaml pyjwt>=2.0.1 pyOpenSSL \
+    'cryptography>=3.3.1,<4.0.0' requests
```

Comment:
optional: if the package requirement is complex, maybe we want to put them into a requirements.txt? Just like what you did for Linux.

Reply:
Done, mostly. I deduplicated one overlapping set of requirements across two build options in this script. `pip install -r ...` is still in the script twice, once for each Objective-C and Python, but I think that's ok for now.